tests/storage/linstor/test_linstor_sr.py

```diff
@@ -18,15 +18,15 @@ class TestLinstorSRCreateDestroy:
     and VM import.
     """
 
-    def test_create_sr_without_linstor(self, host, lvm_disk):
+    def test_create_sr_without_linstor(self, host, lvm_disk, provisioning_type, storage_pool_name):
         # This test must be the first in the series in this module
         assert not host.is_package_installed('python-linstor'), \
             "linstor must not be installed on the host at the beginning of the tests"
         try:
             sr = host.sr_create('linstor', 'LINSTOR-SR-test', {
-                'group-name': STORAGE_POOL_NAME,
-                'redundancy': '1',
-                'provisioning': 'thin'
+                'group-name': storage_pool_name,
+                'redundancy': '2',
```

Comment:
Is there a specific reason to set 2 in your tests?

Reply:
I think we must handle this error in the driver itself. In theory a simple busy loop should fix that.

Reply:
Thanks, will keep it as `1`.